engine.mk

```diff
@@ -177,7 +177,7 @@ endif
 
 #build mode defines
 ifeq ($(USER), user)
-GLOBAL_DEFINES += RAMDUMP_MODE_OFF
+GLOBAL_DEFINES += RAMDUMP_MODE_OFF LK3RD_USER
```

Comment:
How about we move that out to the platform rules.mk? As-in LK3RD_USER=1

Reply:
The debug logger commit was a very-much temporary measure -  but sure, if I decide to merge it.
Acked

Reply:
Commit dropped.